### PR TITLE
Adding captions to images

### PR DESCRIPTION
**Pull request by lqueipom**

### CODE REVIEW

**Comment by Tetheta** [comment]
Starting review

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/images.html [hunk 1]
```diff
@@ -9,21 +9,51 @@
...
+            <div class="caption">Tibidabo, Spain</div></a>
+        <a href="images-me/laura-3.jpg">
+            <img src="images-me/laura-3.jpg"/>
+            <div class="caption">The Bunker, Spain</div></a>
+        <a href="images-me/laura-4.jpg">
+            <img src="images-me/laura-4.jpg"/>
+            <div class="caption">Parc Guell, Spain</div></a>
+        <a href="images-me/laura-5.jpg">
+        <img src="images-me/laura-5.jpg"/>
+        <div class="caption">Turd Salt Mine, Romaina</div></a>
```
Comment (Tetheta) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/images.html:26]: Your indenting here is a bit inconsistent, would be nice to keep it the same throughout.

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/images.html [hunk 2]
```diff
@@ -9,21 +9,51 @@
...
-        <a href="images-me/laura-7.jpg"><img src="images-me/laura-7.jpg"/></a>
-        <a href="images-me/laura-8.jpg"><img src="images-me/laura-8.jpg"/></a>
-        <a href="images-me/laura-9.jpg"><img src="images-me/laura-9.jpg"/></a>
-        <a href="images-me/laura-10.jpg"><img src="images-me/laura-10.jpg"/></a>
-        <a href="images-me/laura-11.jpg"><img src="images-me/laura-11.jpg"/></a>
-        <a href="images-me/laura-12.jpg"><img src="images-me/laura-12.jpg"/></a>
-        <a href="images-me/laura-13.jpg"><img src="images-me/laura-13.jpg"/></a>
-        <a href="images-me/laura-14.jpg"><img src="images-me/laura-14.jpg"/></a>
-	<a href="images-me/laura-15.jpg"><img src="images-me/laura-15.jpg"/></a>
+        <a href="images-me/laura-1.jpg">
```
Comment (Tetheta) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/images.html:12]: So is the goal of the <a> tag here to make the image clickable to show by itself as well as displaying it normally?

Comment (lqueipom) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/images.html:12]: I believe so yes! I wanted to both display the image and make it clickable so you could see the whole picture. Was that what you were referring to? Thank you for the comments!

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/images.html (hunk 1 above)
Comment (lqueipom) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/images.html:26]: You are completely right, did not realize will fix now!

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/images.html (hunk 2 above)
Comment (Tetheta) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/images.html:12]: Yup, sounds good

**Pull request merged**